docs/shared/lve_manager/README.md

```diff
@@ -673,6 +673,11 @@ To modify package limits click on a pencil symbol in <span class="notranslate">_
 
 When limits are set click <span class="notranslate">_Save_</span> to apply changes or <span class="notranslate">_Cancel_</span> to close the window.
 
+
+`VE_DEFAULT` package is not a real package, it's a view for default limits. You can modify default limits on a Users tab.
```

Comment:
:::tip Note
The `VE_DEFAULT` package is not a real package, it's a view for default limits. Limits from this virtual package are applied to a user when no other limit values are applicable. You can modify default limits on the Users tab.

On cPanel, limits of the package named `default` are applied to users with the `default` package assigned - which is, as the name suggests, the package applied by default to new users. This has priority over the `VE_DEFAULT` virtual package mentioned above.
:::

Reply:
fixed